sdk/storage/azure-storage-blobs/CHANGELOG.md

```diff
@@ -62,6 +62,7 @@
 - Replace `std::pair<int64_t, int64_t>` with `Azure::Core::Http::Range` to denote blob ranges.
 - Move version strings into `Details` namespace.
 - Replace scoped enums that don't support bitwise operations with extensible enum.
+- Rename `SourceConditions` in API options to `SourceAccessConditions`.
```

Comment:
Is there a reason to make this source specific?  Could it be AccessConditions instead?
https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.storage.accesscondition?view=azure-dotnet

Reply:
So in .Net, this is called `conditions` for most APIs. And for copy API, they are called `sourceConditions` and `destinationConditions`.

But we think `conditions` is not intuitive enough. So we use `AccessConditions` for destination access conditions, and `SourceAccessCondition` for source access conditions.